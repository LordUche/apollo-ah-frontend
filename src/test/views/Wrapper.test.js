import React from 'react';
import { shallow } from 'enzyme';
import { createMockStore } from '../setup';
import Wrapper from '../../views/common/Wrapper';

createMockStore();

describe('<Wrapper>', () => {
  it('should render without crashing', () => {
    const container = (
      <Wrapper>
        <p>Hello</p>
      </Wrapper>
    );
    const wrapper = shallow(container);
    expect(wrapper).toMatchSnapshot();
  });
});
