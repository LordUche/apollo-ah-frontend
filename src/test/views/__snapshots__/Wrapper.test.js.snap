// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper> should render without crashing 1`] = `ShallowWrapper {}`;
